website/server/models/user/methods.js

```diff
@@ -110,16 +110,17 @@ schema.methods.addComputedStatsToJSONObj = function addComputedStatsToUserJSONOb
 // To negotiate between the payment providers and the payment helper (which probably has too many responsiblities)
 // In summary, currently is is best practice to use this method to cancel a user subscription, rather than calling the
 // payment helper.
-schema.methods.cancelSubscription = async function cancelSubscription () {
+schema.methods.cancelSubscription = async function cancelSubscription (options = {}) {
```

Comment:
Yes, and also tests, but as I said, this is a WIP and incomplete. I was posting it just to see if it's the best approach. I'm guessing though that you think it's okay to proceed with doing it this way?